ch06ex02/test/LoginTest.js

```diff
@@ -1,19 +1,24 @@
 // For usage info, see https://nodejs.org/api/assert.html
-const assert = require("assert");
+const assert = require("assert").strict;
```

Comment:
Enabling strict asserts by default.
Probably just a stop gap. Using **assert** feels a bit cheap, and we should probably eventually  swap it with chai or some other assert engine.